src/color-system.mdx

```diff
@@ -39,7 +39,7 @@ The reduced brightness in dark mode is set with usability in mind, and will help
 ### Using CSS variables
 We have not used HEX values to create the CSS variables for each color. They are instead written in RGB digits, without parentheses. For example `--color-blue-default: 33, 150, 243;` (which would be `#2196f3` in HEX). The reason for such a choice is empowering you to write the colors with alpha channel if needed.
 
-Thus, to get a solid blue color, you would write `color: rgb(var(--color-blue-default));`.  
+Thus, to get a solid blue color, you would write `color: rgb(var(--color-blue-default));`.
```

Comment:
I'm guessing these spaces were removed automatically? If you have your editor set to automatically remove trailing whitespace, you should disable that, and instead make sure you have the editorconfig plugin installed, as the built-in setting doesn't know to make an exception for Markdown, while our editorconfig settings do.

Reply:
(I'll fix the spaces though, so you don't have to commit any fixup 🙂)